index.html

```diff
@@ -0,0 +1,196 @@
+              yields
+            </span>
+          </li>
+
+            <span class="date">1953</span>
+            <span class="date">1953</span>
+            <span class="detail"
+              >crosses a short, sturdy dwarf breed ofwheat with a high-yeidling
+              American breed, creating a strain that responds well to
+              fertilizer. It goes on to provide 95% of Mexico's wheat</span
+            >
+          </li>
+          <li>
+            <span class="date">1962</span> -
+            <span class="detail"
+              >Visits Delhi andrings his high-yielding strains of wheat to the
+              Indian subcontinent in time to help mitigate mass starvation due
+              to a rapidly expanding population
+            </span>
+          </li>
+          <li>
+            <span class="date">1970</span> -
+            <span class="detail">receives the Nobel Peace Prize</span>
+          </li>
+          <li>
+            <span class="date">1983</span> -
+            <span class="detail"
+              >helps seven African countries dramatically increase their maize
+              and sorghum yields
+            </span>
+          </li>
+          <li>
+            <span class="date">1984</span> -
+            <span class="detail"
+              >becomes a distinguished professor at Texas A&M University
+            </span>
+          </li>
+          <li>
+            <span class="date">2005</span> -
+            <span class="detail"
+              >states "we will have to double the world food supply by 2050."
+              Argues that genetically modified crops are the only way we can
+              meet the demand, as we run out of arable land. Says that GM crops
+              are not inherently dangerous because "we've been genetically
+              modifying plants and animals for a long time. Long before we
+              called it science, people were selecting the best breeds."
+            </span>
+          </li>
+          <li>
+            <span class="date">2009</span> -
+            <span class="detail">dies at the age of 95.</span>
+          </li>
+        </ul>
+        <div class="tribute">
+          <p>
+            "Borlaug's life and achievement are testimony to the far-reaching
+            contribution that one man's towering intellect, persistence and
+            scientific vision can make to human peace and progress."
+            <br />
+            <br />
+            <span> -- Indian Prime Minister Manmohan Singh </span>
+          </p>
+        </div>
+      </div>
+      <footer>
+        <h4>
+          If you have time, you should read more about this incredible human
+          being on his
+
+          <a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+            >Wikipedia entry.</a
+          >
+        </h4>
+      </footer>
+    </div>
+  </main>
+=======
```

Comment:
> remove unnecessary wordings

- [ ] You are required to use the `polite` language when making the comments on the code. This enable the contribute to feel relaxed and can clearly understand your request. In addition, i kindly suggest that you give the detailed explanation to `why` you are suggesting for those changes. This will help the contributor to apply the exact changes you referred to ☺️ .